ansible/playbook.yaml

```diff
@@ -29,35 +28,37 @@
       tags: [init-server, node_exporter]
     - role: ethpandaops.general.prometheus
       tags: [init-server, prometheus]
+    - role: ethpandaops.general.vector
+      tags: [init-server, vector]
 
 - hosts: bootnode
   become: true
   roles:
     - role: ethpandaops.general.cl_bootnode
       tags: [cl_bootnode]
-    - role: eth_testnet_config
-      tags: [ethereum, eth_testnet_config]
+    - role: ethpandaops.general.ethereum_testnet_config
+      tags: [ethereum, eth_testnet_config, ethereum_testnet_config]
```

Comment:
why have `eth_testnet_config` and `ethereum_testnet_config`?

Reply:
supporting legacy tag names, which is what you might be used to when used as a local role.